c/zis/services/auth.c

```diff
@@ -90,13 +61,81 @@ static int handleVerifyPassword(AuthServiceParmList *parmList,
 
   FILL_SAF_STATUS(&parmList->safStatus, safRC, racfRC, racfRsn);
   CMS_DEBUG(globalArea, "handleVerifyPassword() done\n");
-  if (idta != NULL) {
-    safeFree(idta, sizeof(IDTA));
-    idta = NULL;
-  }
   return rc;
 }
 
+static int handleGenerateToken(AuthServiceParmList *parmList,
+                                const CrossMemoryServerGlobalArea *globalArea) {
+       if(parmList->safIdtService.safIdtServiceVersion > 1) {
```

Comment:
Don't use magic numbers, define the version in the header via `#define`.

Reply:
added constant